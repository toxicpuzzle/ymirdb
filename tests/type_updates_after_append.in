SET a 1 2 3
SET b 1 2
TYPE a
APPEND a b
TYPE a
bye