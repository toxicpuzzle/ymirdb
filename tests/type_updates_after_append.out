> ok

> ok

> simple

> ok

> general

> bye
